packages/generator/src/imports.ts

```diff
@@ -59,15 +62,17 @@ export function corePropertyTypeImportNames(
 export function corePropertyFieldTypeImportNames(
   properties: VdmProperty[]
 ): string[] {
-  return unique(
-    properties.filter(prop => !prop.isComplex).map(prop => prop.fieldType)
-  );
+  return unique([
+    ...properties.filter(prop => !prop.isComplex).map(prop => prop.fieldType),
+    ...(properties.some(prop => prop.isMulti) ? ['CollectionField'] : [])
```

Comment:
I find the name `isMulti` a bit missleading. Sounds like multi link or so. If isMulti releates to collection Field yes/no then I would call it: `isCollectionField`

Reply:
I was actually considering calling this `isCollection` and using it for both navigation props and props.